e2e/create-pvc_test.go

```diff
@@ -81,197 +81,224 @@ var _ = Describe("CreatePVC", func() {
 		_ = os.RemoveAll(tmpDir)
 	})
 
-	It("creates a pvc and attaches to pod", Label("pvcCreation"), func() {
-		pvcName := "test-pvc"
-		storageClassName := "kubevirt"
-		pvc := pvcSpec(pvcName, storageClassName, "1Gi")
+	Context("Filesystem volume mode", func() {
+		It("creates a pvc and attaches to pod", Label("pvcCreation"), func() {
```

Comment:
Many of our existing tests seem to make sense for both block and filesystem pvcs. How practical is it to begin expanding our existing tests to be tables, which execute an entry for filesyste and block.

if this makes sense, I don't think you necessary have to transition all the tests to work for block and filesystem right away. But starting with a small subset could make sense which we expand on later

Reply:
Should be relatively straight forward, we could pass in a function that builds the pvcs and pods and keep the rest of the logic identical. Note that our tests should test things specific to our provisioner, the k8s e2e test make sure we do the proper things from a k8s perspective. From that perspective block or filesystem should not matter that much to our provisioner.